src/repositories/actions/postgres/actions.ts

```diff
@@ -0,0 +1,19 @@
+import { BaseTypeORMRepo } from '@rebaseitlabs/typeorm-generic';
+import { v4 as uuidv4 } from 'uuid';
+
+import { PgActionEntity } from '../../../models/entities/pg-actions';
+import { Action } from '../../../models/types/role';
+
+export default class ActionsRepo extends BaseTypeORMRepo<PgActionEntity> {
```

Comment:
Lets add a ActionsInterface (which will driectly extend the BaseInterface from the package)
Then write here that we are implementing the ActionsInterface